.travis.yml

```diff
@@ -2,3 +2,5 @@ language: node_js
 
 node_js:
   - '6'
+  - '4'
+  - '0.10'
```

Comment:
I wouldn't call it hostile, but if it's helpful for you, and there are no problems for me - why not?